src/Symfony/Bundle/TwigBundle/Resources/config/twig.xml

```diff
@@ -165,5 +166,14 @@
             <argument type="service" id="http_kernel" />
             <argument>%twig.exception_listener.controller%</argument>
         </service>
+
+        <service id="twig.configurator.environment" class="Symfony\Bundle\TwigBundle\DependencyInjection\Configurator\EnvironmentConfigurator">
```

Comment:
We didn't pay enough attention to such things so far. But to be correct it would be required.